filters/filter_19_Annoyances_Popups/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 19,
+  "name": "AdGuard Popups filter",
+  "description": "Blocks in-page pop-ups on web pages. Already included in AdGuard Annoyances filter.",
```

Comment:
pop-ups and inline subscriptions?

Reply:
I'm not sure about adding this. Yes, we have inline subscriptions, but also popups.